### DISCUSSION TITLE
AVOID / PATCH problem on Patched TG's

### DISCUSSION THREAD

**Discussion by pezcat19** [chuot/rdio-scanner#435] [Q&A]
Hi all, 

I have an issue where on my local P25 Trunked system, where there are "Patched" TG's. When I am listening to the various TG traffic and I want to AVOID some of these TG's, I have found that pressing the the AVOID button on these "Patched" TG's causes the display to show "PATCH"  (30/60/120M) and not "AVOID".  When it displays "PATCH", I continue to hear the TG's as normal. There seems to be no way to AVOID these Patched TG's? 

Even if I see the real TG ID for a patched TG is 30001, and I manually de-selecting this TG in the "SELECT TG" screen, it makes no difference. 

Am I doing something wrong? Or is this a bug? I just need to AVOID the TG traffic on PATCHED TG's as required, by using (avoiding) the common TGID number (like 30001). 

It seems like the AVOID feature is not simply avoiding a straight TGID number? (which is what I would like). It seems to be avoiding some other metric? 

Thanks in advance for any help.

**Comment by doftheworld** [comment]
Little late to the party, but the answer is that you have not avoided whatever talkgroups are patched to it. When talkgroups are patched, they act as one - so you need to have all the patched talkgroups avoided, or you will still hear the one you have avoided. 

Sometimes PD in my area patch random talkgroups and it takes me ages to figure out what they are patched to.
